### ISSUE TITLE
Devcontainer not starting (since vscode v1.64.0 ?)

### ISSUE THREAD

**Issue by fortierq** [qdm12/latexdevcontainer#24]
Since I upgraded vscode to the last version, it looks like this devcontainer is not working anymore. I get this error:
```
Error response from daemon: Container cfdc96daad3fa4591d3bb8ac8b5361557437fa5812cfa696f00d936b4cca37f6 is not running
```
 And, in the container logs:
```
Container started
zsh:1: unmatched "
```

**Comment by qdm12** [comment]
Fixed by #25 thanks for reporting & fixing it!

**Comment by qdm12** [comment]
Just a thank you again, that 'bug' they introduced in vscode v1.64.0 was really a pain just this morning with another development container. Fixed it using your fix.
